### PR TITLE
Accept Query and Mutation root names as args

### PR DESCRIPTION
**Pull request by emilianoLeite**
Greetings from Brazil! 🇧🇷  👋 

I tried using your script to generate a postman collection, but the script failed since my `Query` and `Mutation` root fields had names different from `RootQueryType` and `RootMutationType`, respectively.

Therefore I added these changes so the Query and Mutation root field names can be parameterized

### CODE REVIEW

File: src/exporters/insomnia/insomnia.ts [hunk 1]
```diff
@@ -3,18 +3,18 @@ import { Root, RequestGroup, Request, RequestBody } from './types';
...
         let graphQLRequestsGroup = new RequestGroup(url);
         rootExport.addResource(graphQLRequestsGroup);
+
         types.forEach(type => {
-            if (type.name == "RootQueryType" || type.name == "RootMutationType") {
+            if (type.name == rootQueryName || type.name == rootMutationName) {
                 let requestGroup = new RequestGroup(type.name)
                 requestGroup.parentId = graphQLRequestsGroup._id;
                 rootExport.addResource(requestGroup);
-                let query: any;
```
Comment (emilianoLeite) [inline src/exporters/insomnia/insomnia.ts:17]: `type` and  `query` were not being used

File: src/exporters/insomnia/insomnia.ts [hunk 2]
```diff
@@ -27,12 +27,12 @@ class Insomnia {
                         returnFields = this.buildReturnFields(returnType);
                     }
 
-                    if (query.type.ofType != null) {
+                    if (query.type.ofType != null && query.type.ofType.kind !== "LIST") {
```
Comment (emilianoLeite) [inline src/exporters/insomnia/insomnia.ts:30]: From what I could infer from [the docs](https://graphql.org/learn/introspection/), the `LIST` kind does not have a name, so entering this if branch was causing the script to crash in such cases

File: src/index.ts [hunk 3]
```diff
@@ -36,4 +46,30 @@ const argv = yargs
...
+        chalk.cyan(
+            'Argument' + chalk.italic(' mutation-name ') +
+            'was not supplied. Using default value: ' +
+            chalk.italic.bgCyan.black(` ${defaultValue} `)
+        )
+    )
+    argv['mutation-name'] = defaultValue
+}
+
+convert(argv.url, argv.format, argv['query-name'], argv['mutation-name'], argv['custom-headers']);
```
Comment (oshosanya) [inline src/index.ts]: We can try to maintain consistency on this line in terms of how object properties are accessed, as some parts use the dot notation and the other use the 'array index' notation

**Comment by oshosanya** [comment]
Thanks a lot for the PR, already tried working on this, but this is much better than what I would have produced.

File: src/index.ts (hunk 3 above)
Comment (emilianoLeite) [reply src/index.ts]: Yeah, good point. I think the easiest way to achieve consistency is moving every to 'index' notation
```suggestion
convert(argv['url'], argv['format'], argv['query-name'], argv['mutation-name'], argv['custom-headers']);
```

What do you think?

**Comment by emilianoLeite** [comment]
> Thanks a lot for the PR, already tried working on this, but this is much better than what I would have produced.

Thanks 😄  I also want to praise you for the code architecture, it was really easy to debug it and understand what each function was doing!

**Comment by emilianoLeite** [comment]
As a side note, would you mind including this repo in the [Hacktoberfest event](https://hacktoberfest.digitalocean.com/)? I would really appreciate it if you did 😄  ! 

To include this repo you can: 
- Add the `hacktoberfest` topic to the repo **OR**
- Add a `hacktoberfest-accepted` to the Pull Requests

More info can be found [here](https://hacktoberfest.digitalocean.com/hacktoberfest-update)

File: src/index.ts (hunk 3 above)
Comment (oshosanya) [reply src/index.ts]: Great

**Comment by oshosanya** [comment]
> As a side note, would you mind including this repo in the [Hacktoberfest event](https://hacktoberfest.digitalocean.com/)? I would really appreciate it if you did 😄 !
> 
> To include this repo you can:
> 
> * Add the `hacktoberfest` topic to the repo **OR**
> * Add a `hacktoberfest-accepted` to the Pull Requests
> 
> More info can be found [here](https://hacktoberfest.digitalocean.com/hacktoberfest-update)

Done, Thanks. Package published to npm.

**Comment by gidox1** [comment]
Great work, man!
Open issues, let's contribute!

**Pull request merged**